data/map.txt

```diff
@@ -8979,7 +8979,7 @@ system "Devil-Hide"
 	trade Medical 784
 	trade Metal 345
 	trade Plastic 481
-	fleet "Devil-Run Warships" 400
```

Comment:
I found this could get pretty overwhelming about 1/3 of the time unless it was at least 1000. You'll sometimes get a few heavy warships in at the same time, and they'll focus all fire on one escort.

With your changes to the tribute fleet though, I'll have to replay to see if it is enough.

Reply:
I made the warship fleet sometimes contain firebirds and tweaked others spawnings as well